packages/babel-helper-module-transforms/src/normalize-and-load-metadata.js

```diff
@@ -249,6 +254,25 @@ function getModuleMetadata(programPath: NodePath, loose: boolean) {
     }
   }
 
+  for (const [source, metadata] of sourceData) {
+    if (
+      lazy !== false &&
+      !(isSideEffectImport(metadata) || metadata.reexportAll)
+    ) {
+      if (lazy === true) {
+        // 'true' means that local relative files are eagerly loaded and
+        // dependency modules are loaded lazily.
+        metadata.lazy = !/\./.test(source);
+      } else if (Array.isArray(lazy)) {
+        metadata.lazy = lazy.indexOf(source);
+      } else if (typeof lazy === "function") {
+        metadata.lazy = lazy(source);
+      } else {
+        throw new Error(`'lazy' must be a boolean, string array, or function`);
```

Comment:
Here it shold throw the same message as https://github.com/babel/babel/pull/6952/files#diff-71f62b3225be3bae0ddc0c8e49189aa7R29

Reply:
Actualy, can this error be thrown? It should already been throw at https://github.com/babel/babel/pull/6952/files#diff-71f62b3225be3bae0ddc0c8e49189aa7R29

Reply:
One throws up front when you declare the plugin, which seemed like it would be best for users, this one throws inside the transform itself in case someone else decides to use `helper-module-transforms` and calls it wrong. I can make the errors the same though, that's true.